Libraries/src/Amazon.Lambda.Serialization.Json/AwsResolver.cs

```diff
@@ -86,7 +86,7 @@ protected override IList<JsonProperty> CreateProperties(Type type, MemberSeriali
                     }
                 }
             }
-            else if (type.FullName.Equals("Amazon.DynamoDBv2.Model.StreamRecord", StringComparison.Ordinal))
+            else if (type.FullName.Equals("Amazon.Lambda.DynamoDBEvents.DynamoDBEvent+StreamRecord", StringComparison.Ordinal))

```

Comment:
Is there a reason we can't leave the old `else if`? That way if somebody updates `Amazon.Lambda.Serialization.Json` but not the event package they wouldn't get a break in behavior.

Reply:
No, that's a good call out; I did not realize this was a separate package. I've updated this `if` to include both type names in: https://github.com/aws/aws-lambda-dotnet/commit/3bd10bc4900e4d21ebba4d47fe9700cd0076a74a#diff-ba736be4021bfb4dcd5cb24447e5ba3396f9bf3b18b6f32a2407101383c22e72R89